xarray/tests/test_backends.py

```diff
@@ -609,8 +609,8 @@ def test_orthogonal_indexing(self):
             assert_identical(expected, actual)
 
     @pytest.mark.xfail(
-        not has_dask,
-        reason="the code for indexing without dask handles negative steps in slices incorrectly",
+        has_dask,
+        reason="the code for indexing with dask handles negative steps in slices incorrectly",
```

Comment:
This is strange, and goes opposite to the statement at https://github.com/pydata/xarray/pull/3919#issuecomment-606267473 :confused:

- Before: `test_vectorized_indexing` would *fail* when there is **no dask**.
- Now: the test *fails* **with dask**, but* passes* when there is **no dask**.